build.yaml

```diff
@@ -14,7 +14,7 @@ cassandra:
   - 2.1
   - 2.2
   - 3.0
-  - 3.10
+  - 3.7
```

Comment:
This was giving some errors so I'm doing that in a separate branch: https://github.com/datastax/python-driver/tree/python-update-buildyaml